### PR TITLE
docs: add 1.29 language port release notes

### PR DESCRIPTION
**Pull request by mxschmitt**

### CODE REVIEW

File: docs/src/release-notes-csharp.md [hunk 1]
```diff
@@ -4,6 +4,49 @@ title: "Release notes"
...
+    await Page.RouteAsync("**/api/settings", async route => {
+      // Fetch original settings.
+      var response = await route.FetchAsync();
+
+      // Force settings theme to a predefined value.
+      var json = await response.JsonAsync();
+      json["theme"] = "Solorized";
+
+      // Fulfill with modified data.
+      await route.FulfillAsync(new RouteFulfillOptions {
```
Comment (mxschmitt) [inline docs/src/release-notes-csharp.md:23]:
```suggestion
      await route.FulfillAsync(new(){
```

File: docs/src/release-notes-csharp.md [hunk 2]
```diff
@@ -4,6 +4,49 @@ title: "Release notes"
...
+        Json = json
+      });
+    });
+    ```
+
+- New method [`method: Locator.all`] to iterate over all matching elements:
+
+  ```csharp
+  // Check all checkboxes!
+  var checkboxes = await Page.LocatorAsync("role=checkbox");
```
Comment (mxschmitt) [inline docs/src/release-notes-csharp.md:33]:
```suggestion
  var checkboxes = Page.Locator("role=checkbox");
```

File: docs/src/release-notes-java.md [hunk 3]
```diff
@@ -4,6 +4,30 @@ title: "Release notes"
...
 
+## Version 1.29
+
+### New APIs
+
+- New method [`method: Locator.all`] to iterate over all matching elements:
+
+    ```java
+    // Check all checkboxes!
+    const checkboxes = page.getByRole('checkbox');
```
Comment (mxschmitt) [inline docs/src/release-notes-java.md:15]:
```suggestion
    Locator checkboxes = page.getByRole('checkbox');
```

**Review by pavelfeldman — APPROVED** [review]
Let's add new selectOption semantics everywhere?

File: docs/src/release-notes-csharp.md [hunk 4]
```diff
@@ -4,6 +4,63 @@ title: "Release notes"
...
+        Json = json
+      });
+    });
+    ```
+
+- New method [`method: Locator.all`] to iterate over all matching elements:
+
+  ```csharp
+  // Check all checkboxes!
+  var checkboxes = Page.Locator("role=checkbox");
```
Comment (yury-s) [inline docs/src/release-notes-csharp.md:33]:
```suggestion
  var checkboxes = Page.GetByRole(AriaRole.CHECKBOX);
```

File: docs/src/release-notes-java.md [hunk 5]
```diff
@@ -4,6 +4,44 @@ title: "Release notes"
...
 
+## Version 1.29
+
+### New APIs
+
+- New method [`method: Locator.all`] to iterate over all matching elements:
+
+    ```java
+    // Check all checkboxes!
+    Locator checkboxes = page.getByRole('checkbox');
```
Comment (yury-s) [inline docs/src/release-notes-java.md:15]:
```suggestion
    Locator checkboxes = page.getByRole(AriaRole.CHECKBOX);
```

**Review by yury-s — APPROVED** [review]

**Pull request merged**